src/app/clinic/clinic_facade.cpp

```diff
@@ -0,0 +1,45 @@
+#include "clinic_facade.hpp"
+
+void Clinic::appendDoctor(std::shared_ptr<Doctor> doctor)
+{
+    doctors_.push_back(doctor);
```

Comment:
I have created an issue for that. [Issue #62](https://github.com/coders-school/desktop-business-app/issues/62)
It would be good to validate it within createDoctor() method and keep Clinic class clean of such

Reply:
Place of putting validation to be further considered. It's hard to decide where is best instantly IMO